FloripaTrans/FTBaseViewController.h

```diff
@@ -0,0 +1,14 @@
+//
+//  FTBaseViewController.h
+//  FloripaTrans
+//
+//  Created by Vítor Makoto Matayoshi de Moraes on 3/6/15.
+//  Copyright (c) 2015 Vitor Makoto. All rights reserved.
+//
+#import <UIKit/UIKit.h>
+
+@interface FTBaseViewController : UIViewController
+- (void)dismissKeyboard;
+- (void)alertFailFetchingMessage:(NSString *)message;
+- (void)alertMessage:(NSString *)message withTilte:(NSString *)title;
```

Comment:
Oh, thanks for spotting it. I just fixed on a1b21e2326024d656cffe01b445c684423e33925.